spec/factories/courses.rb

```diff
@@ -47,6 +47,15 @@
     passcode 'pizza'
   end
 
+  factory :basic_course, class: 'BasicCourse' do
```

Comment:
This doesn't create a save-able course. Passcode, school and term are part of the validation.

https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/app/models/course.rb#L118
https://github.com/WikiEducationFoundation/WikiEduDashboard/blob/master/app/models/course.rb#L306

This is relevant for the follow-on work with #662